services/QuillLMS/app/models/user.rb

```diff
@@ -142,6 +142,8 @@ class User < ApplicationRecord
   has_many :user_subscriptions
   has_many :subscriptions, through: :user_subscriptions
   has_many :activity_sessions
+  has_many :started_activities, through: :activity_sessions, source: :activity
```

Comment:
Move this from the student concern since it needs to be defined after activity_sessions when used in user. There's probably more cleanup with redundant associations, but I was trying to be as light touch as possible.